jupyter_client/tests/test_multikernelmanager.py

```diff
@@ -463,10 +449,10 @@ async def test_subclass_callables(self):
         mkm.get_kernel(kid).reset_counts()
         mkm.reset_counts()
         await mkm.shutdown_all(now=True)
-        assert mkm.call_count('shutdown_kernel') == 1
+        #assert mkm.call_count('shutdown_kernel') == 1
```

Comment:
Per the previous comment, I think enabling these commented-out assertions is the way to go.